packages/cli/src/cmds/account.ts

```diff
@@ -138,14 +144,16 @@ class AccountLogin extends YargsCommand {
     });
     switch (args.service) {
       case "azure": {
-        const result = await outputAzureInfo("login");
+        await AzureTokenProvider.signout();
```

Comment:
Is this to make sure every time login command will open login page?

Reply:
Yes, this is for Azure login, should we change M365 login scenario to this?
@chagong @swatDong @Alive-Fish

Reply:
I think so. And the change is already there.

Reply:
Very nice.